scripts/capstone.py

```diff
@@ -103,20 +137,32 @@ class CapstoneDisassembleCommand(GenericCommand):
     def __init__(self) -> None:
         super().__init__(complete=gdb.COMPLETE_LOCATION)
         gef.config[f"{self._cmdline_}.use-capstone"] = GefSetting(
-            False, bool, "Replace the GDB disassembler in the `context` with Capstone", hooks={"on_write": self.switch_disassembler})
-        self.__original_disassembler: Optional[Callable[[
-            int, int, Any], Generator[Instruction, None, None]]] = None
+            False,
+            bool,
+            "Replace the GDB disassembler in the `context` with Capstone",
+            hooks={
+                "on_write": [
+                    self.switch_disassembler,
+                ]
+            },
+        )
+        self.__original_disassembler: Optional[
+            Callable[[int, int, Any], Generator[Instruction, None, None]]
+        ] = None
         return
 
     def switch_disassembler(self, _) -> None:
         ctx = gef.gdb.commands["context"]
         assert isinstance(ctx, ContextCommand)
-        if gef.config[f"{self._cmdline_}.use-capstone"] == True:
+        if gef.config[f"{self._cmdline_}.use-capstone"]:
             self.__original_disassembler = ctx.instruction_iterator
             ctx.instruction_iterator = cs_disassemble
         else:
             # `use-capstone` set to False
-            if ctx.instruction_iterator == cs_disassemble and self.__original_disassembler:
+            if (
```

Comment:
Do you prefer this style (four lines) over either 1 line or 2 with parens or 2 with `\`?

Reply:
I almost forgot: I started using `black` for gef-extras (only for now), and those kind of output were the result. I'm not sure how I feel yet about it.

Reply:
Ah yeah, that may be why I didn't like black (remember I said it was too opinonated and I didn't like the opinion).

I am 100% for auto-formatting everything! That will be nice and easy to check. I don't love black but I could live with it.